### PR TITLE
Fix in get_typed_nodes()

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/node.py [hunk 1]
```diff
@@ -55,9 +55,10 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
 
     """
     nodes = get_node_objs()
+    from pdb import set_trace; set_trace()
```
Comment (prsurve) [inline ocs_ci/ocs/node.py]: do we need set_trace here? it will need user input to proceed

Comment (ebenahar) [reply ocs_ci/ocs/node.py]: no :)

File: ocs_ci/ocs/node.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
     """
     nodes = get_node_objs()
```
Comment (MeridianExplorer) [inline ocs_ci/ocs/node.py]: I believe we can skip the assignment and use get_nodes_objs()` inline in the list comprehension below

**Comment by MeridianExplorer** [comment]
LGTM, it's great that we're improving readability of the code, very important in my opinion

File: ocs_ci/ocs/node.py (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/ocs/node.py]: fixed

**Pull request merged**